web/styles/subscriptions.css

```diff
@@ -365,8 +371,7 @@ h4.user_group_setting_subsection_title {
 
     .left .no-streams-to-show,
     .left .no-groups-to-show,
-    .right .nothing-selected .create-stream-button-container,
-    .right .nothing-selected .create-group-button-container {
```

Comment:
It didn't feel worth it to have it clubbed together for two properties, so just extracted it out.

Reply:
yeah, it's better this way.